rllib/agents/dqn/dqn_policy.py

```diff
@@ -180,7 +151,11 @@ def build_q_model(policy, obs_space, action_space, config):
         v_min=config["v_min"],
         v_max=config["v_max"],
         sigma0=config["sigma0"],
-        parameter_noise=config["parameter_noise"])
+        # TODO(sven): Move option to add LayerNorm after each Dense
+        #  generically into ModelCatalog.
+        add_layer_norm=isinstance(
+            getattr(policy, "exploration", None), ParameterNoise)
+        or config["exploration_config"]["type"] == "ParameterNoise")
```

Comment:
Is the config check sufficient, or do we also need to check the policy exploration attribute type?

Reply:
The config should be sufficient, unless a user decides to configure the exploration via an already instantiated object (which is possible). This will be removed once we move layernorm as a setting into the Catalog.